app/views/provider_mailer/organisation_permissions_set_up.text.erb

```diff
@@ -0,0 +1,28 @@
+Dear <%= @provider_user.full_name %>
+
+# <%= @partner_organisation.name %> has set up organisation permissions for teacher training courses you work on with them
+
+You can now manage applications made through GOV.UK for the courses you work on with <%= @partner_organisation.name %>.
+
+They have set up the following organisation permissions.
+
+<%= I18n.t('provider_relationship_permissions.make_decisions.description') %>:
+
+<%= "- #{@recipient_organisation.name}" if @permissions.permit?(:make_decisions, @recipient_organisation) %>
+<%= "- #{@partner_organisation.name}" if @permissions.permit?(:make_decisions, @partner_organisation) %>
+
```

Comment:
Sorry to trickle in comments, but we can put this bit in a loop over the permissions